libs/datamodel/core/tests/validation/attributes/schema/mysql_enums_do_not_have_a_schema.prisma

```diff
@@ -0,0 +1,25 @@
+datasource testds {
+    provider = "mysql"
+    url = env("TEST_DATABASE_URL")
+    schemas = ["public", "security", "users"]
+}
+
+generator js {
+    provider = "prisma-client-js"
+    previewFeatures = ["multiSchema"]
+}
+
+
+enum MyEnum {
+    ONE
+    TWO
+
+    @@schema("users")
+}
+
+// [1;91merror[0m: [1mMySQL enums do not belong to a schema.[0m
+//   [1;94m-->[0m  [4mschema.prisma:17[0m
+// [1;94m   | [0m
+// [1;94m16 | [0m
+// [1;94m17 | [0m    @@schema([1;91m"users"[0m)
```

Comment:
the error would be better if we highlighted the whole attribute, I'll try to get to it in a follow up